src/schism/schism_nuopc_cap.F90

```diff
@@ -650,30 +671,28 @@ subroutine SetClock(comp, rc)
 !> @todo (not registered and fully implemented yet, so not used)
 subroutine SetRunClock(comp, rc)
 
-  use schism_bmi, only : schismTimeStep
-
   type(ESMF_GridComp)  :: comp
   integer, intent(out) :: rc
 
-  type(ESMF_Clock)              :: clock
-  type(ESMF_TimeInterval)       :: timeStep
-  integer(ESMF_KIND_I4)         :: localrc
-  real(ESMF_KIND_R8)            :: seconds
+  type(ESMF_Clock) :: dclock, mclock
```

Comment:
dclock and mclock should have better descriptive names

Reply:
@platipodium it is done.